SportsHub/SportsHub.Application/Services/ArticleService.cs

```diff
@@ -99,14 +99,22 @@ public async Task<List<Article>> GetListOfArticlesBySubstringAsync(string substr
         public async Task DeleteArticleAsync(int id)
         {
             var articleForDelete = await _unitOfWork.ArticleRepository.GetByIdAsync(id);
-            
+
             if (articleForDelete is null)
             {
                 throw new NotFoundException(string.Format(ExceptionMessages.NotFound, ExceptionMessages.Article));
             }
-            
+
+            var userHasArticle = await _unitOfWork.UserRepository.DeleteArticle(articleForDelete);
+
+            if (userHasArticle is false)
```

Comment:
Better to use `!userHasArticle` than `is` operator.

Reply:
fix this too and I will merge